Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.cpp

```diff
@@ -246,6 +254,35 @@ namespace EMStudio
         m_timeViewToolBar->UpdateInterface();
 
         EMotionFX::AnimGraphEditorNotificationBus::Handler::BusConnect();
+
+        // Create the motion event properties widget.
+        m_motionEventWidget = new MotionEventWidget();
+        m_motionEventWidget->hide();
+        connect(this, &TimeViewPlugin::SelectionChanged, this, [=]
```

Comment:
suggestion/question: Are we sure we want the default capture clause to be by value? Sometimes it's clearer to mention the actual values to be used explicitly to make it clear what is captured. It's a style thing but I usually go for this, up to you on what makes most sense